### DISCUSSION TITLE
About Runtime Bindings API in Unity

### DISCUSSION THREAD

**Discussion by 0xF6** [LibraStack/UnityMvvmToolkit#58] [General]
https://forum.unity.com/threads/introducing-the-runtime-bindings-api-in-unity-2023-2.1454221/

What do you think about this?

**Comment by ChebanovDD** [comment]
It was anticipated that the Unity team would eventually incorporate data binding into the `UI Toolkit`. The only question remains whether they can offer a more convenient and functional solution.

In the meantime, the Unity team is working on their solution, I suggest using mine. 😉

P.S. Pending approval, the `UnityMvvmToolkit` will soon be available on the `Unity Asset Store`.
